t5/data/mixtures.py

```diff
@@ -25,24 +25,49 @@
 from t5.data.utils import rate_unsupervised
 import tensorflow_datasets as tfds
 
-
+# These weights are based on the number of examples in each dataset.
+_SUPER_GLUE_WEIGHT_MAPPING = {
+    "dpr_v001_simple": 1_322.,
+    "super_glue_wsc_v102_simple_train": 554.,
```

Comment:
This number is based on the number of examples before preprocessing. Training examples from WSC get dropped when the target is not True (https://github.com/google-research/text-to-text-transfer-transformer/blob/master/t5/data/preprocessors.py#L1388). The correct number of examples here (after preprocessing) should be 259.